server/src/main/java/org/apache/druid/server/coordinator/RandomBalancerStrategy.java

```diff
@@ -28,20 +28,22 @@
 import java.util.NavigableSet;
 import java.util.Set;
 import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
 
 public class RandomBalancerStrategy implements BalancerStrategy
 {
   @Override
   public ServerHolder findNewSegmentHomeReplicator(DataSegment proposalSegment, List<ServerHolder> serverHolders)
```

Comment:
minor nit:mark return type as nullable

Reply:
It is marked as Nullable in super class `BalancerStrategy`